components/net/http_loader.rs

```diff
@@ -1984,7 +1970,7 @@ async fn http_network_fetch(
     let meta_headers = meta.headers;
     let cancellation_listener = context.cancellation_listener.clone();
     if cancellation_listener.cancelled() {
-        return Response::network_error(NetworkError::Internal("Fetch aborted".into()));
+        return Response::network_error(NetworkError::ConnectionFailure);
```

Comment:
Huh, this error should be LoadCancelled instead. That variant is new since this code was written.